python/pynessie/client/_endpoints.py

```diff
@@ -310,12 +310,14 @@ def merge(
     :param merge_json: merge content
     :param expected_hash: expected hash of HEAD of branch
     :param ssl_verify: ignore ssl errors if False
+    :return: json dict of a merge response
     """
     url = "/trees/branch/{}/merge".format(branch)
     params = {}
     if expected_hash:
         params["expectedHash"] = expected_hash
-    _post(base_url + url, auth, json=merge_json, ssl_verify=ssl_verify, params=params)
+    ret_dict = cast(dict, _post(base_url + url, auth, json=merge_json, ssl_verify=ssl_verify, params=params))
```

Comment:
nit: lets not do too much on a single line
instead have:
```
response = _post(...)
return cast(dict, response)
```

Reply:
updated